### PR TITLE
Boolean option for domain structure

### LINKED ISSUE #14: Add option to whether or not show domain structure
The input image for XitoSBMLcli could be more than 1 image. Thus, if we have 100 images as input, the domain structure of 100 images will be generated which could be annoying.
So, we should introduce a new boolean option to allow users to choose if they want to see the domain structure during their run.

### PR DESCRIPTION
**Pull request by Medha-B**
Regarding issue https://github.com/Medha-B/XitoSBML/issues/14

### CODE REVIEW

File: src/main/java/jp/ac/keio/bio/fun/xitosbml/cli/CliRun.java [hunk 1]
```diff
@@ -30,6 +30,8 @@
 	String inputValue;
 	@Option(names = "-o", required = true, description = "The path to output XML file if input is image file")
 	String outputValue;
+	@Option(names = "-ds", description = "To show the domain hierarchy structure. False by default.")
```
Comment (kaitoii11) [inline src/main/java/jp/ac/keio/bio/fun/xitosbml/cli/CliRun.java]: It is customary to define an option with a single hyphen with a single letter and double hyphen with multiple letters. For example, the help option is defined as `-h` and `--help`.

So we should follow that convention.

**Review by kaitoii11 — CHANGES_REQUESTED** [review]

File: src/main/java/jp/ac/keio/bio/fun/xitosbml/cli/CliRun.java (hunk 1 above)
Comment (Medha-B) [reply src/main/java/jp/ac/keio/bio/fun/xitosbml/cli/CliRun.java]: Ok. Will do so.

**Comment by Medha-B** [comment]
Is it correct now?

**Review by kaitoii11 — APPROVED** [review]
lgtm
But it says that there is a conflict so we should fix it before merging

**Comment by Medha-B** [comment]
> lgtm
> But it says that there is a conflict so we should fix it before merging

Resolved conflicts.

**Pull request merged**